src/cli/argument-parser.ts

```diff
@@ -260,8 +260,7 @@ export default class CLIArgumentParser {
     }
 
     private _setUpProviderName (): void {
-        if (this.opts.listBrowsers)
-            this.opts.providerName = this.opts.listBrowsers as string;
+        this.opts.providerName = (this.opts.listBrowsers === true ? void 0 : this.opts.listBrowsers) as string;
```

Comment:
I don't remember the use case for which this code is necessary.
If you can call to mind the use case, please write a test for that.
If not, then just remove the `_setUpProvideName` function.

Reply:
You can run `testcafe --list-browsers` to get locally installed browsers or `testcafe --list-browsers providerName` to get browsers supported by a `providerName` provider.

In the first case, `listBrowsers` is `true`, in the second - `"providerName"`.

Let's just write a test for the expected behaviour, and then just specify a default value for the `listBrowsers` option: https://github.com/tj/commander.js/#default-option-value